src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.targets

```diff
@@ -37,6 +37,10 @@
     <IlcSetupPropertiesDependsOn>ImportRuntimeIlcPackageTarget</IlcSetupPropertiesDependsOn>
     <IlcDynamicBuildPropertyDependencies>SetupProperties</IlcDynamicBuildPropertyDependencies>
 
+    <!-- NativeAOT apps do not require deps.json and runtimeconfig.json files -->
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
```

Comment:
Test?

Reply:
I created an issue for adding a test in the sdk repo for this, https://github.com/dotnet/sdk/issues/26132